core/scripts/pptr-run-devtools.js

```diff
@@ -145,6 +145,9 @@ function addSniffer(receiver, methodName, override) {
 }
 
 async function waitForLighthouseReady() {
+  // @ts-expect-error global
```

Comment:
Leave a comment about benefit of undocking early? If this function gets rearranged at some point, this isn't the kind of line that folks typically check the blame for and your testing knowledge will be lost.